src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck1 = new Truck();
+        Machine buldozer1 = new Bulldozer();
+        Machine excavator1 = new Excavator();
```

Comment:
don't put a number as a part of the name 
Be careful with typo mistakes 
```suggestion
        Machine truck = new Truck();
        Machine bulldozer = new Bulldozer();
        Machine excavator = new Excavator();
```